examples/shapes/05-logo-raylib-anim.lisp

```diff
@@ -0,0 +1,96 @@
+                                :color *faded-black*))
+               (white-rec (make-rectangle (- (/ (get-screen-width) 2) 112)
+                                          (- (/ (get-screen-height) 2) 112)
+                                          224 224
+                                          *faded-white*))
+               (replay-text (make-text "[R] REPLAY" 340 200 :size 20 :color +gray+)))))
+
+(defun main ()
+  (with-window (:title "raylib [shapes] example - raylib logo animation")
+    (with-scenes *scene*
+      (with-scene-objects (top-side-rec left-side-rec bottom-side-rec right-side-rec text white-rec
+                                        replay-text) *scene*
+          (do-game-loop (:livesupport t
+                         :vars ((frames-counter 0)
+                                (letters-count 0)
+                                (state 0)
+                                (alpha 1.0)))
+            (case state
+              ;; Small box blinking
+              (0 (progn (incf frames-counter)
+                        (when (= frames-counter 120) (setf frames-counter 0
+                                                           state 1))))
+              ;; Top and left bars growing
+              (1 (progn (incf (width top-side-rec) 4)
+                        (incf (height left-side-rec) 4)
+                        (when (= (width top-side-rec) 256) (setf state 2))))
+              ;; Bottom and right bars growing
+              (2 (progn (incf (width bottom-side-rec) 4)
+                        (incf (height right-side-rec) 4)
+                        (when (= (width bottom-side-rec) 256) (setf state 3))))
+              ;; Letters appearing (one by one)
+              (3 (progn (incf frames-counter)
+                        (when (= (rem frames-counter 12) 0)
+                          (incf letters-count)
+                          (setf frames-counter 0))
+                        (when (>= letters-count 10)
+                          (decf alpha 0.02)
+                          (when (<= alpha 0) (setf alpha 0
+                                                   state 4)))
+                        (setf (text text) (substring 0 letters-count "raylib"))))
+              ;; Reset and Replay
+              (4 (when (is-key-pressed-p +key-r+)
+                   (setf frames-counter 0
+                         letters-count 0
+                         (width top-side-rec) 16
+                         (height left-side-rec) 16
+                         (width bottom-side-rec) 16
+                         (height right-side-rec) 16
+                         (text text) ""
+                         alpha 1.0
+                         state 0))))
+
+            ;; Change the colour fading
+            (fade *faded-black* alpha)
+            (fade *faded-white* alpha)
+
+            (with-drawing
+              (case state
+                (0 (when (= 0 (mod (truncate frames-counter 15) 2))
```

Comment:
Also, does this not work via `draw-scene`?

Reply:
Errr... the lines below this, not above it. GitHub confuses me sometimes.

Reply:
Yeah `draw-scene` totally works. I it was nice to have a plural for `draw-object` (handy when an object is already in scope). If you don't like that, I'll scrap it. If you do like it, we could add it to claylib itself.

Reply:
Yeah, `draw-objects` is fine. I forgot we're basically already doing that anyway, just with a scene. You can add that into Claylib proper and resubmit.

Reply:
Done (https://github.com/defun-games/claylib/pull/21/commits/1634cb88b27ed1147367e772f3947424d58cd2c5)! Should be good to merge.